src/interpreter/runtime/castIntoStackLocalOfType.js

```diff
@@ -9,15 +9,26 @@ const f64 = require("./values/f64");
 export function castIntoStackLocalOfType(
   type: string,
   v: any,
-  nan: boolean = false
+  nan: boolean = false,
+  inf: boolean = false
 ): StackLocal {
   const castFn = {
     i32: i32.createValueFromAST,
     i64: i64.createValueFromAST,
-    f32: nan ? f32.createNanFromAST : f32.createValueFromAST,
+    f32: f32.createValueFromAST,
     f64: f64.createValueFromAST
   };
 
+  if (nan) {
+    castFn.f32 = f32.createNanFromAST;
+    castFn.f64 = f64.createNanFromAST;
```

Comment:
I noticed that `f64.createNanFromAST` doesn't exist.

Reply:
I added the placeholders, but there is no `nan` code for `f64` yet.